Appsilon/rhino
# chore: Upgrade Cypress to 13.0.0.

Upgrade Cypress to v13.

# How to test:
1. Initialize a new Rhino application. Make sure that you have Rhino v1.5.0.9000 installed both globally and in the application.
2. Run Cypress tests with `rhino::test_e2e(interactive = TRUE)`. Make sure that it uses Cypress 13
3. Run `rhino::test_e2e(interactive = FALSE)`
